hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -114,6 +114,7 @@
     }
     {
       name: stub_cpu_mode
+      sw_images: ["sw/device/boot_rom/boot_rom:0"]
```

Comment:
In stub CPU mode, the CPU is not running any code. This should not be required.

Reply:
Wait, I may have said something in haste - I guess this is for the ROM integrity stuff? It will be good to understand this change better.